Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -761,7 +761,7 @@ def _on_resume_err(self, failure):
         failure.trap(CancelledError)
         self._logger.error("Resume data cancelled")
 
-    @waitForHandleAndSynchronize()
+    @checkHandleAndSynchronize(default=Deferred().callback(None))
```

Comment:
Thanks for the input! 

After I looked up into it, `defer.succeed(None)` also works as I wanted. Is that preferred method for this case?